Sim/Acdata/f16bk15.dat

```diff
@@ -1463,7 +1463,7 @@ avionicsCanUseSpice 0
 avionicsJDAMAvionicsType 0
 avionicsDiffICPStyle 0
 avionicsMLUPFL 0
-avionicsDatalink 1
+avionicsDatalink 3

```

Comment:
Fairly certain the >2 value to disable DL, as per https://www.benchmarksims.org/forum/showthread.php?32206-How-to-Disable-Data-Link-Is-it-Possible&highlight=avionicsDatalink doesn't work - I will merge anyway in hopes BMS team future fixes it.

Reply:
I will repost I just posted in #Kuwait-modding:

> Btw, I just tested how to disable datalink:
> "avionicsDatalink 3" does NOT disable DL,
> "avionicsDatalink 1" disables DL,
> "avionicsDatalink 0" enables DL.